src/platforms/linux/daemon/dbusservice.cpp

```diff
@@ -172,19 +180,48 @@ void DBusService::userListCompleted(QDBusPendingCallWatcher* watcher) {
   if (reply.isValid()) {
     UserDataList list = reply.value();
     for (const auto& user : list) {
-      m_appTracker->userCreated(user.userid, user.path);
+      userCreated(user.userid, user.path);
```

Comment:
Why is `m_appTracker->` removed here?

Reply:
The gist of this PR is that we use a timer to poll the runtime state of the user's session and delay the app-tracker setup until the state settles, however, I only wanted to write that polling logic once so I put it into `DBusService::userCreated()` and made `DBusService::userListCompleted()` call that instead of invoking `m_appTracker->userCreated()` directly.

Note that we have two cases where we update the set of active users. When starting the daemon we fetch the list of running users by asking systemd via the `UserList` method on D-Bus, and then as new users are created and destroyed we get changes to the list of active users from the `UserNew` and `UserRemoved` signals.

Reply:
Makes sense, thanks for explaining this